cardano-explorer-db/src/Explorer/DB/Migration.hs

```diff
@@ -8,37 +8,41 @@ module Explorer.DB.Migration
   , runMigrations
   ) where
 
-import           Control.Exception (SomeException, bracket, handle)
-import           Control.Monad (forM_, unless)
-import           Control.Monad.IO.Class (liftIO)
-import           Control.Monad.Logger (NoLoggingT)
-import           Control.Monad.Trans.Reader (ReaderT)
-import           Control.Monad.Trans.Resource (runResourceT)
-
-import           Data.Conduit.Binary (sinkHandle)
-import           Data.Conduit.Process (sourceCmdWithConsumer)
-import           Data.Either (partitionEithers)
-import qualified Data.List as List
-import qualified Data.ByteString.Char8 as BS
-import           Data.Text (Text)
-import qualified Data.Text as Text
-import qualified Data.Text.IO as Text
-import           Data.Time.Clock (getCurrentTime)
-import           Data.Time.Format (defaultTimeLocale, formatTime, iso8601DateFormat)
-
-import           Database.Persist.Sql (SqlBackend, SqlPersistT, entityVal, getMigration, selectFirst)
+import           Control.Exception             (SomeException, bracket, handle)
+import           Control.Monad                 (forM_, unless)
+import           Control.Monad.IO.Class        (liftIO)
+import           Control.Monad.Logger          (NoLoggingT)
+import           Control.Monad.Trans.Reader    (ReaderT)
+import           Control.Monad.Trans.Resource  (runResourceT)
+
+import qualified Data.ByteString.Char8         as BS
+import           Data.Conduit.Binary           (sinkHandle)
+import           Data.Conduit.Process          (sourceCmdWithConsumer)
+import           Data.Either                   (partitionEithers)
+import qualified Data.List                     as List
+import           Data.Text                     (Text)
+import qualified Data.Text                     as Text
+import qualified Data.Text.IO                  as Text
+import           Data.Time.Clock               (getCurrentTime)
+import           Data.Time.Format              (defaultTimeLocale, formatTime,
+                                                iso8601DateFormat)
+
+import           Database.Persist.Sql          (SqlBackend, SqlPersistT,
+                                                entityVal, getMigration,
+                                                selectFirst)
```

Comment:
Can we reverse this bit? Aligning stuff like this always causes endless amounts of commit noise.